### ISSUE TITLE
[Bug]: JASP crashes after opening .txt

### ISSUE THREAD

**Issue by AnguloB** [jasp-stats/jasp-issues#1603]
### JASP Version

0.16

### What analysis are you seeing the problem on?

All 

### What OS are you seeing the problem on?

Windows 10

### Bug Description

After opening a .txt file, which apparently has no problem, when trying to do any analysis (descriptive, cfa, etc.) the program closes suddenly. The data have been created with R, and can be opened in R, jamovi and excel. 

### Expected Behaviour

To the analysis 

### Steps to Reproduce

1. Import File
2. Click in any part of the menu
[JASP 2022-02-04 14_19_40 Desktop.log](https://github.com/jasp-stats/jasp-issues/files/8003310/JASP.2022-02-04.14_19_40.Desktop.log)
[JASP 2022-02-04 14_19_40 Engine 0.log](https://github.com/jasp-stats/jasp-issues/files/8003311/JASP.2022-02-04.14_19_40.Engine.0.log)
[JASP 2022-02-04 14_13_42 Desktop.log](https://github.com/jasp-stats/jasp-issues/files/8003312/JASP.2022-02-04.14_13_42.Desktop.log)
[JASP 2022-02-04 14_13_42 Engine 0.log](https://github.com/jasp-stats/jasp-issues/files/8003313/JASP.2022-02-04.14_13_42.Engine.0.log)
[JASP 2022-02-04 14_14_09 Desktop.log](https://github.com/jasp-stats/jasp-issues/files/8003314/JASP.2022-02-04.14_14_09.Desktop.log)
[JASP 2022-02-04 14_14_09 Engine 0.log](https://github.com/jasp-stats/jasp-issues/files/8003315/JASP.2022-02-04.14_14_09.Engine.0.log)
[JASP 2022-02-04 14_14_35 Desktop.log](https://github.com/jasp-stats/jasp-issues/files/8003316/JASP.2022-02-04.14_14_35.Desktop.log)
[JASP 2022-02-04 14_14_35 Engine 0.log](https://github.com/jasp-stats/jasp-issues/files/8003317/JASP.2022-02-04.14_14_35.Engine.0.log)
[Dades_20220204.txt](https://github.com/jasp-stats/jasp-issues/files/8003319/Dades_20220204.txt)

### Log (if any)

```bash
(logs and data  attached in previous step)
```

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if any.
- [X] I have included a JASP file that causes the crash/bug, if any.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by JorisGoosen** [comment]
Hey @AnguloB 

Does this only happen for this particular datafile?
Also, does JASP crash immediately after adding the analysis?
Or do you first have to select some columns?
Is it possible to run a filter or add a computed column without an analysis?

And you say it closes suddenly? So no error msg or anything? No warning box?

**Comment by AnguloB** [comment]
I have tried with other dataset with same structure different numbers and also happens. It crashes when you try to use the menu, not before.

**Comment by JorisGoosen** [comment]
> when you try to use the menu

Aka when you click in the ribbonbar?

But what about  my other questions:
> Is it possible to run a filter or add a computed column without an analysis?

> And you say it closes suddenly? So no error msg or anything? No warning box?

**Comment by AnguloB** [comment]
Yes, when I click on the ribbonbar.  It is possible to do the "syntax" of the filter, but not to run it. 

There is no message or anything. My feeling is that it tries to do something, freezes, and then suddenly closes. 

Now I have managed to start a confirmatory factor analysis, but when I reopened the data it went wrong again.

**Comment by JorisGoosen** [comment]
And if you just start JASP, not load any data but open a "Summary Statistics" analysis?
Or "Distributions" or something.

Can you then run those?

**Comment by AnguloB** [comment]
Thank you very much for your patience. Normally I would not insist so much but it is important as I am planning to use it with my students next semester.

In case it might help, the data has been generated in R with lavaan's simulateData function. Exported as txt with write.table().

There is no problem with running those you said. Days before I have worked with other txt files (maybe with less variables) with no problems.

**Comment by JorisGoosen** [comment]
No problemo!

I still don't have an idea what exactly could be going wrong but maybe the latest development version: https://static.jasp-stats.org/Nightlies/WindowsZip/JASP-nightly-development-cfb5fe3123038592ab8500bda48d6214cc4f3451.zip

Doesnt crash? This is what we're currently testing for the upcoming 0.16.1 release. You can use it to test but there might still be bugs in it we haven't found.

**Comment by JorisGoosen** [comment]
Also, maybe you can try switching this on:
![image](https://user-images.githubusercontent.com/29062713/152655416-3f35853c-a032-4f12-81e4-37ddde3e2a49.png)

Not sure if it helps, but your system is in spanish no?

**Comment by AnguloB** [comment]
Yes, I use the Spanish system. Switching the native encoding has not improved the situation.

The nightly version seems stable for the moment, I have made several tests with the dataset and it works correctly.

**Comment by JorisGoosen** [comment]
Well, that's great! 
Whatever the problem is we apparently already fixed so I guess this is solved.

The JASP you now have is probably very similar if not the same as what we'll be releasing soon.
So I think you could already prepare with this. But I would advise to at least open and refresh the files in the real 0.16.1 once it's out to be sure.

**Comment by AnguloB** [comment]
Hello again Joris, 
I was able to do everything I wanted to do with the nightly version you proposed me.  However, a colleague of mine is doing the same analysis with the nightly version and the program keeps freezing. When this happens the only option she has is to close the program and start over (if she has not been lucky enough to save the little bit of progress). 

On the other hand, the results screen often freezes and does not allow you to see the new outputs.  She is using also use a windows 10.

This is probably a different issue but we have detected some problems in the translation of some concepts. For example the intraclass correlation coefficient in the "reliability" module is translated as "partial correlation". Is there any place where we can report this kind of things?

As always, thank you very much for your advance. 

Translated with www.DeepL.com/Translator (free version)
